eitprocessing/parameters/tidal_impedance_variation.py

```diff
@@ -0,0 +1,194 @@
+import itertools
+from dataclasses import dataclass, field
+from functools import singledispatchmethod
+from typing import Literal
+
+import numpy as np
+
+from eitprocessing.datahandling.breath import Breath
+from eitprocessing.datahandling.continuousdata import ContinuousData
+from eitprocessing.datahandling.eitdata import EITData
+from eitprocessing.datahandling.intervaldata import IntervalData
+from eitprocessing.datahandling.sequence import Sequence
+from eitprocessing.features.breath_detection import BreathDetection
+from eitprocessing.features.pixel_breath import PixelBreath
+from eitprocessing.parameters import ParameterCalculation
+
+
+@dataclass
+class TIV(ParameterCalculation):
+    """Compute the tidal impedance variation (TIV) per breath."""
+
+    method: Literal["extremes"] = "extremes"
+    breath_detection_kwargs: dict = field(default_factory=dict)
+
+    def __post_init__(self) -> None:
+        if self.method != "extremes":
+            msg = f"Method {self.method} is not implemented. The method must be 'extremes'."
+            raise NotImplementedError(msg)
+
+    @singledispatchmethod
+    def compute_parameter(
+        self,
+        data: ContinuousData | EITData,
+    ) -> str:
```

Comment:
A single dispatch-method should not have a return type. This function never returns anything. The overloaded versions (compute_continuous_parameter and compute_pixel_parameter) should have proper return types.